tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -632,6 +636,11 @@ unsigned int TTreeProcessorMT::GetMaxTasksPerFilePerWorker()
    return fgMaxTasksPerFilePerWorker;
 }
 
+unsigned int TTreeProcessorMT::GetTasksPerWorker()
+{
+   return fgTasksPerWorker;
```

Comment:
Those accessor are not thread safe per se.  Is that intentional?

Reply:
This logic is unchanged from previously at least.

Typical use case would be to call this together with EnableImplicitMT()

Reply:
@pcanal I don't think it's reasonable for users to expect that calling a class-level setter is thread-safe, unless explicitly documented. Feel free to open a GitHub "improvement" if you would like these to be made thread-safe.

Reply:
That is fair to say.  Whether the user will then do the right thing (never ever call this after enabling threads) is a question. (i.e. there is no actual way for the user to ensure that when they call any of those (preexisting and new) Set function that some other threads (for example triggered has part of an experiment framework) is not calling the Getter (and possibly get a very bad value).